scripts/presubmit.sh

```diff
@@ -93,16 +93,27 @@ main() {
     go build ${goflags} ${go_dirs}
 
     echo 'running go test'
-    echo "" > coverage.txt
 
+    # Individual package profiles are written to "profile_$i.out" files under
+    # /tmp/trillian_profile.
+    # An aggregate profile is created at /tmp/coverage.txt.
+    mkdir -p /tmp/trillian_profile
+    rm -f /tmp/trillian_profile/*
+
+    i=0
```

Comment:
idle thought: rather than having a second loop variable, is there an easy way to convert `$d` into a flat name (no /) and have (say) `/tmp/trillian_profile/profile_util_etcd.out

Reply:
Done.

It's a bit trickier than the i logic, but it gives better file names, so I guess it's a win overall.